views/js/apipCreator/api/apipItem.js

```diff
@@ -0,0 +1,212 @@
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technologies SA ;
+ *
+ */
+define([
+    'lodash',
+    'jquery',
+    'taoQtiItem/apipCreator/helper/parser',
+    'taoQtiItem/apipCreator/helper/serializer',
+    'taoQtiItem/apipCreator/api/qtiElement',
+    'taoQtiItem/apipCreator/api/accessElement',
+    'taoQtiItem/apipCreator/helper/jquery.xpath.min'
+], function (_, $, parser, serializer, QtiElement, AccessElement) {
+    'use strict';
+    
+    /**
+     * Instanciate an creator api that will works on an APIP authoring model
+     * 
+     * @param {String} apipItemXML - the APIP-QTI item XML
+     * @returns {Object}
+     */
+    function ApipItem(apipItemXML) {
+        this.apipDoc = parser.parse(apipItemXML);
+        this.$apipDoc = $(this.apipDoc);
+        this.XMLNS = {
+            'apip': 'http://www.imsglobal.org/xsd/apip/apipv1p0/imsapip_qtiv1p0',
+            'qti': 'http://www.imsglobal.org/xsd/apip/apipv1p0/qtiitem/imsqti_v2p2'
```

Comment:
just a question : would it matter if the qti xml contains namespaced elements such as <m:math>, <xi:include> or <pci:portableCustomInteraction> ?